yal/content_feedback_survey.py

```diff
@@ -19,7 +21,7 @@ async def state_process_content_feedback_trigger(self):
         # Mirror the message here, for response and error handling
         choices = [
             Choice("yes", self._("Yes, thanks!"), additional_keywords=["yes"]),
-            Choice("no", self._("Nope")),
+            Choice("no", self._("Not really")),
```

Comment:
Due to this change you need to go and update the CONTENT_FEEDBACK_KEYWORDS in main.py